solutions/swb-ui/infrastructure/src/SWBUIStack.ts

```diff
@@ -90,6 +91,41 @@ export class SWBUIStack extends Stack {
     const distribution = this._createDistribution(bucket);
     this._deployS3BucketAndInvalidateDistribution(bucket, distribution);
     this._addCognitoURLOutput();
+
+    const customBucketDeploymentNode = this.node.findChild(
+      'Custom::CDKBucketDeployment8693BB64968944B69AAFB0CC9EB8756C'
+    );
+    let metadataNode = customBucketDeploymentNode.node.defaultChild as CfnResource;
+    metadataNode.addMetadata('cfn_nag', {
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      rules_to_suppress: [
+        {
+          id: 'W58',
+          reason: 'TODO:triage Lambda functions require permission to write CloudWatch Logs'
```

Comment:
When CDK creates Lambda resources by default or even if you create Lambda with no roles attached to it. By default it create a default role with `AWSLambdaBasicExecutionRole` policy  and this policy has the access to write to Cloudwatch Logs:

> {
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "logs:CreateLogGroup",
                "logs:CreateLogStream",
                "logs:PutLogEvents"
            ],
            "Resource": "*"
        }
    ]
}

Reply:
so in this case for the reason you can mention `This function has AWSLambdaBasicExecutionRole policy attached which has the write access to Cloudwatch Logs`. Please validate if the 'AWSLambdaBasicExecutionRole' policy is attached, which it should be because this is the default behavior when creating Lambda with CDK.